.github/workflows/velox_docker.yml

```diff
@@ -494,13 +494,74 @@ jobs:
           GLUTEN_IT_JVM_ARGS=-Xmx5G sbin/gluten-it.sh queries-compare \
             --local --preset=velox-with-celeborn --benchmark-type=ds --error-on-memleak --off-heap-size=10g -s=1.0 --threads=8 --iterations=1
 
+  build-native-lib-centos-8:
```

Comment:
Why centos-8 is needed?

Reply:
@zhztheplayer, it's just for distinguishing it with the other static build job for centos-7. We originally planned to just use one cache, produced by static build, but there still exist some issues. So we can firstly keep the two build jobs to produce libraries cache for tpc test jobs (on centos-7) & spark test jobs (on centos-8) respectively.